config/ion_auth.php

```diff
@@ -58,9 +58,9 @@
  | -------------------------------------------------------------------------
  | Bcrypt is available in PHP 5.3+
  | Argon2 is available in PHP 7.2
+ | Argon2id is available in PHP 7.3
  |
- | Argon2 is recommended by expert (it is actually the winner of the Password Hashing Competition
- | for more information see https://password-hashing.net). So if you can (PHP 7.2), go for it.
+ | Argon2id is the current PHP language default.
```

Comment:
I'm not sure that it's true that Argon2id is the default. In my PHP 7.4 environment, bcrypt is still the `PASSWORD_DEFAULT`